README.md

```diff
@@ -67,6 +67,22 @@ Type: `Object`
 
 Default: `{}`
 
+### Port Copying
+This service will automatically launch the standalone server on the port specified in your WebDriverIO configuration file when the `usePortInStandalone` flag is `true`. 
+
+For example:
+```js
+port: 4441,
+usePortInStandalone: true,
```

Comment:
Why adding this extra option? Can't we just always apply the port number (or everytime when it is not `4444`)?

Reply:
I thought at first this was why the CI build was failing (maybe it was looking for the server being up on a specific port), but now I see that's not the case. I guess I'm not sure in what situation you would want the standalone server to be running on a different port than the tests, so I'll take it off.

Reply:
tests are failing because Array.includes is not supported with the node version running in CI/CD